src/Native/Runtime/unix/PalRedhawkInline.h

```diff
@@ -89,3 +89,9 @@ FORCEINLINE void PalMemoryBarrier()
 }
 
 #define PalDebugBreak() __builtin_trap()
+
+extern "C" int __stdcall CoreLibNative_GetLastErrNo();
+FORCEINLINE int PalGetLastError()
+{
+    return CoreLibNative_GetLastErrNo();
```

Comment:
Redhawk PAL should not be calling into CoreLibNative. The Redhawk PAL methods should be calling the underlying OS APIs directly.